### ISSUE TITLE
Perhaps a better visual indicator that you are viewing inside a level's group?

### ISSUE THREAD

**Issue by ladlon** [turtletooth/tahoma2d#347]
When dealing with a level/column that contains several grouped selections of several shapes, it's a little disorienting understanding if you are looking at the root level of the column, or are viewing within one of the groups.

Granted the other items (outside the currently entered group) are greyed out (which is good), but perhaps it's a bit subtle overall.

I'm not sure exactly what a better solution is... other than maybe increasing the greying out effect a bit more, and maybe some visual indicator (red frame around the viewer?) to show you are not at the root level?

Maybe also useful, would be a window you could open that lists all the shapes, showing if they are grouped (kind of like a text-based heirarchy display).  I may have actualy suggested that in a previous Feature Request... but the visual indicator of being inside a group is a new suggestion, though.

**Comment by DarrenTAnims** [comment]
You could show textual breadcrumbs on the viewer titlebar, with each one being a link, so you can click on them to move up the hierarchy: Root > Group > Group > Group

**Comment by ladlon** [comment]
Yes, along with the viewport frame (and perhaps more noticable greying/dimming of the other (non-target) items, that would be a great addition.  I think Moho does that very thing.  Great suggestion.
